client/.swagger-codegen-ignore

```diff
@@ -0,0 +1,2 @@
+**
```

Comment:
Think about `// TODO! Maybe get started by copy/pasting from data-catalog?` until you know what this does

Reply:
Update: tells the codegen not to override anything in these directories. Seems like a good idea to leave it.